### PR TITLE
Prepare release r1.2

### LINKED ISSUE #20: First release for Device Swap v0.1
**Problem description**
Deliver first release for Device Swap API.

cc: @jgarciahospital 

**Expected action**
In my understanding of the release process (from [here](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14559630/API+Release+Process#API-release-numbering)) we can craft a first release r1.1 The version of the API will be v0.1.0-rc

to be check with @camaraproject/release-management_codeowners @hdamker 

**Additional context**
This will be done only when all other relevant issue are solved

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:
Prepare the release 1.2 with the required changes.

#### Which issue(s) this PR fixes:

Fixes #20

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,52 @@
...
+This release contains the definition and documentation of
+* device-swap v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.1
+
+## device-swap v0.1.0
+
+**device-swap v0.1.0 is the 1st release of the version 0.1.0**
```
Comment (hdamker) [inline CHANGELOG.md:22]: There will be only one release of this version

```suggestion
**device-swap v0.1.0 is the public release of the version 0.1.0**
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,52 @@
...
+
+- 0.1.0 Device Swap API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml)
```
Comment (hdamker) [inline CHANGELOG.md:27]: These links won't work with the "blob"

```suggestion
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.2/code/API_definitions/device-swap.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.2/code/API_definitions/device-swap.yaml)
```

NOTE: The same change has to be done within r1.1, the links for Redoc and Swagger there are not working.

File: README.md [hunk 3]
```diff
@@ -19,11 +19,14 @@ Repository to describe, develop, document and test the Device Swap API family
...
-* The pre-release r1.1 contains the version 0.1.0-rc.1 of the Device Swap API. It is the first release candidate of the v0.1.0. The release tag is [r1.1](https://github.com/camaraproject/DeviceSwap/tree/r1.1).
-- 0.1.0-rc.1 Device Swap API definition **with inline documentation**:
-  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceSwap/blob/r1.1/code/API_definitions/device-swap.yaml)
-  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.1/code/API_definitions/device-swap.yaml&nocors)
-  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.1/code/API_definitions/device-swap.yaml)
+* The release r1.2 contains the version 0.1.0 of the Device Swap API. It is the release of the v0.1.0. The release tag is [r1.2](https://github.com/camaraproject/DeviceSwap/tree/r1.2).
```
Comment (hdamker) [inline README.md:22]: Second sentence looks like a duplicate to me:

```suggestion
* The release r1.2 contains the version 0.1.0 of the Device Swap API. The release tag is [r1.2](https://github.com/camaraproject/DeviceSwap/tree/r1.2).
```

**Review by hdamker — CHANGES_REQUESTED** [review]

**Comment by hdamker** [comment]
@jgarciahospital one needed correction and two suggestions, see above

**Comment by jgarciahospital** [comment]
Thanks @hdamker, correction included (also in r1.1) and suggestions merged.

**Review by bigludo7 — APPROVED** [review]
LGTM !

**Pull request merged**